### PR TITLE
[System test] Modify test for happy liquidation path - add collateral tokens to the asset pool

### PR DESCRIPTION
**Pull request by tomaszslabon**
This pull request modifies the happy path liquidation system test to check if collateral tokens from the asset pool will be seized in the correct proportion.
After this pull request is merged, the system test will have all the assertions listed in point 1 of [this table](https://github.com/keep-network/coverage-pools/issues/83#issuecomment-860604097).

### CODE REVIEW

File: test/system/liquidation.test.js [hunk 1]
```diff
@@ -131,18 +144,33 @@ describeFn("System -- liquidation", () => {
...
+        bondedAmount.mul(85).div(100)
+      )
+    })
+
+    it("should transfer collateral tokens to the bidder", async () => {
+      expect(await collateralToken.balanceOf(bidder.address)).to.be.closeTo(
+        to1e18(6), // 30% of the initial asset pool
+        precision
+      )
+      expect(await collateralToken.balanceOf(assetPool.address)).to.be.closeTo(
```
Comment (dimpar) [inline test/system/liquidation.test.js:163]: How about extracting this assertion to a separate `it`? Ex. it("should adjust asset pool's collateral tokens after the claim")

File: test/system/liquidation.test.js [hunk 2]
```diff
@@ -31,10 +32,11 @@ describeFn("System -- liquidation", () => {
   const lotSize = to1e18(5)
   // signers have bonded 290.81 ETH
   const bondedAmount = BigNumber.from("290810391624000000000")
-  // 75% of the deposit is exposed on auction in the liquidation moment
-  const bondedAmountPercentage = BigNumber.from("75")
```
[the remark this answers is not in the document]
Comment (tomaszslabon) [reply test/system/liquidation.test.js:35]: I added `bondedAmountPercentage` back, but changed the description a little.

File: test/system/liquidation.test.js (hunk 1 above)
Comment (tomaszslabon) [reply test/system/liquidation.test.js:167]: I extracted it to a separate `it`, as suggested.

Comment (tomaszslabon) [reply test/system/liquidation.test.js:167]: I extracted it to a separate it, as suggested.

File: test/system/liquidation.test.js [hunk 3]
```diff
@@ -31,10 +32,11 @@ describeFn("System -- liquidation", () => {
...
+  // amount of collateral deposited to asset pool is 20 KEEPS
+  const collateralAmount = to1e18(20)
```
[the remark this answers is not in the document]
Comment (tomaszslabon) [reply test/system/liquidation.test.js]: I replaced it with 200k KEEP tokens and changed the comment.

**Review by dimpar — APPROVED** [review]

**Pull request merged**